runtime_builders/publish_builders.py

```diff
@@ -63,9 +63,8 @@ def _publish_latest(builder_dir):
         project_name = config['project']
 
         parts = os.path.splitext(latest)
-        project_prefix = os.path.join(builder_util.RUNTIME_BUCKET_PREFIX,
-                                      project_name, '-')
-        latest_file = parts[0][len(project_prefix):]
+        prefix = builder_util.RUNTIME_BUCKET_PREFIX + project_name + '-'
```

Comment:
what is the desired output here? the old way makes it `prefix/project_name/-` and the new way makes it `prefixproject_name-`